components/Table/Table.tsx

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import React from 'react';
+import { cn } from '@/utils/utils';
+
+const Table = React.forwardRef<
+  HTMLTableElement,
+  React.HTMLAttributes<HTMLTableElement> & { className?: string }
+>(({ className, ...props }, ref) => (
+  <table
+    ref={ref}
+    className={cn(
+      'w-[1136px] h-[336px] border rounded-[6px] border-datatableborder',
+      className,
+    )}
+    {...props}
+  />
+));
+Table.displayName = 'Table';
+
+const TableHeader = React.forwardRef<
+  HTMLTableSectionElement,
+  React.HTMLAttributes<HTMLTableSectionElement> & { className?: string }
+>(({ className, ...props }, ref) => (
+  <thead
+    ref={ref}
+    className={cn(
+      'w-[1136px] h-[48px] border-b border-datatableborder text-left',
+      className,
+    )}
+    {...props}
+  />
+));
+TableHeader.displayName = 'TableHeader';
+
+const TableFooter = React.forwardRef<
+  HTMLTableSectionElement,
+  React.HTMLAttributes<HTMLTableSectionElement> & { className?: string }
+>(({ className, ...props }, ref) => (
+  <tfoot
+    ref={ref}
+    className={cn(
+      'border-t bg-muted/50 font-medium [&>tr]:last:border-b-0',
+      className,
+    )}
+    {...props}
+  />
+));
+TableFooter.displayName = 'TableFooter';
+
+const TableRow = React.forwardRef<
+  HTMLTableRowElement,
+  React.HTMLAttributes<HTMLTableRowElement> & { className?: string }
+>(({ className, ...props }, ref) => (
+  <tr
+    ref={ref}
+    className={cn(
+      'w-[1136px] h-[48px] border-b border-datatableborder text-left',
+      className,
+    )}
+    {...props}
+  />
+));
+TableRow.displayName = 'TableRow';
+
+const TableHead = React.forwardRef<
+  HTMLTableCellElement,
+  React.ThHTMLAttributes<HTMLTableCellElement> & { className?: string }
+>(({ className, ...props }, ref) => (
+  <th
+    ref={ref}
+    className={cn(
+      'w-[352px] h-12 border-b border-datatableborder px-3 py-2 gap-2 font-semibold text-sm text-datatableheader',
```

Comment:
And more custom classes using pixels. PIxels are awful to use for width and height, you should always go for a responsive unit.